content/coop.html

```diff
@@ -311,23 +312,22 @@ <h3> Gender </h3>
   Women are paid less than men on average for co-op terms.
   </span>
 
-  Although Women are paid less than men, this is not necessarily related to a gender bias. Some
-  possible reasons could be different interests in terms of work location, type of work, etc among
+  Although women are paid less than men, this is not necessarily related to a gender bias. Some
+  possible reasons could be due to different interests in terms of work location, type of work, etc. among
```

Comment:
I would rather reword this to say "it is difficult to determine what portion of the compensation difference is due to gender vias vs. student interest etc."

Currently, it sounds too dismissive of gender-based income discrimination, even though I know we make this clear in the following paragraph.